cloud.ks

```diff
@@ -16,7 +16,9 @@ services --disabled=network,avahi-daemon
 
 zerombr
 clearpart --initlabel --all
-bootloader --timeout=1 --append="no_timer_check console=tty1 console=ttyS0,115200n8 net.ifnames=0 biosdevname=0"
+# Add "ip=dhcp rd.neednet=1 enforcing=0 coreos.first_boot" for ignition for now
+# Add "coreos.oem.id=ec2" for ec2 target for now
```

Comment:
Looks like ``coreos.oem.id=ec2`` is also added here. No biggie. If you want to add it to the above line we'll wait to merge, but it's not a blocker.

Reply:
not sure what you mean exactly - i broke the two comments out on purpose but maybe I should break each arg out and give a comment about each one. will do that now.